Dockerfile

```diff
@@ -1,4 +1,38 @@
-FROM amd64/node:10.16.3@sha256:925162bdc4b23422763cb2d08ecf1661bbbd856ab54b959517924054e17813fc AS tsbuild
+# Base image
+#============
+FROM renovate/yarn:1.19.1@sha256:9f034a5962dc0ad3212934346f179503cc29045f2551f03a359637bd79151134 AS base
+
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>"
+LABEL name="renovate"
+LABEL org.opencontainers.image.source="https://github.com/renovatebot/renovate"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+
+WORKDIR /usr/src/app/
+
+# required for install
+USER root
+
+RUN chown -R ubuntu:ubuntu /usr/src/app/
```

Comment:
I'm not sure this achieves anything here, unless we built renovate/yarn badly?

Reply:
maybe we no longer need to write to this path?

Reply:
OK let's remove this section and see if everything is still fine